pkg/identity/identity.go

```diff
@@ -218,7 +218,15 @@ func LookupReservedIdentityByLabels(lbls labels.Labels) *Identity {
 		return identity
 	}
 
+	// Cases:
+	//   1) host
+	//   2) remote-node
+	//   3) kube-apiserver
+	//   4) host + kube-apiserver
+	//   5) remote-node + kube-apiserver
+
```

Comment:
I think this is an artifact from the discussions Chris and I had, in order to think about all the possible combinations of labels based on the ordering of events that we receive. I'll take another look and see if I can come up with a reasonable explanation of how to interpret this with the switch case below, or otherwise I'll just drop it.